src/components/Player.js

```diff
@@ -86,12 +95,14 @@ function Player() {
             alt="player2"
             style={
               current
-                ? { cursor: "pointer" }
+              ? { cursor: "pointer" }
                 : { opacity: 0.3, pointerEvents: "none" }
             }
             onClick={() => {
               interactCall("setPlayers", "player",  "DT", "");
               setPlayers(current);
+              localStorage.setItem("score1", 0);
```

Comment:
use proper naming conventions for score1 - its a player 1 score not score1

Reply:
No I did'nt added this. Score1 and Score2 name is given by Swayamjit himself

Reply:
I am just reseting the Score = 0. when user click on that button